2022q2/gcc.adoc

```diff
@@ -12,7 +12,7 @@ Contact: Piotr Kubaj <pkubaj@FreeBSD.org>
  * salvadore@ worked on the upgrade of GCC_DEFAULT in Mk/bsd.default-versions.mk from 10 to 11, opening bug reports based on antoine@'s exp-runs and fixing some: many thanks to all those that helped with this task. The GCC_DEFAULT update from GCC 10 to GCC 11 has now been committed by gerald@ and happened in time for the next quarterly branch.
    link:https://bugs.freebsd.org/bugzilla/show_bug.cgi?id=258378[https://bugs.freebsd.org/bugzilla/show_bug.cgi?id=258378]
 
- * pkubaj@ switched GCC bootstrapping to use Link Time Optimization for GCC itself for GCC 11 and newer by introducing a new option enabled by default. Building with LTO_BOOTSTRAP enabled requires significant amounts of memory and time. How much resources are actually needed depends on your configuration (e.g. are you building from ports or with poudriere? What is your architecture?). To give an idea, link:https://bugs.freebsd.org/bugzilla/show_bug.cgi?id=264949#c17[a user reported that he needed 5 GiB of tmpfs]. Consider disabling LTO_BOOTSTRAP in favor of STANDARD_BOOTSTRAP (or disabling BOOTSTRAP altogether) in case that is a problem.
+ * pkubaj@ switched GCC bootstrapping to use Link Time Optimization for GCC itself for GCC 11 and newer by introducing a new option enabled by default. Building with LTO_BOOTSTRAP enabled requires significant amounts of memory and time. How much resources are actually needed depends on your configuration (e.g. are you building from ports or with poudriere? What is your architecture?). To give an idea, link:https://bugs.freebsd.org/bugzilla/show_bug.cgi?id=264949#c17[a user reported that he needed 5 GiB of tmpfs], while link:https://bugs.freebsd.org/bugzilla/show_bug.cgi?id=265254#c2[another one estimated he would need about 130GB of memory for a successful build]. Consider disabling LTO_BOOTSTRAP in favor of STANDARD_BOOTSTRAP (or disabling BOOTSTRAP altogether) in case that is a problem.
```

Comment:
s/another one estimated he would need/another estimated they would need/

Reply:
Is the use of singular they really needed here? While I am not giving the name of the reporter, the reporter's identity is well known, I don't think this case needs a gender neutral form.

Moreover, if gender neutrality is needed, I usually prefer to use passive form if possible (in this case I would use "but in a different case it has also been estimated that 130GB of memory would be needed for a successful build"), but this might be because I am not a native English speaker and I will not insist if you confirm that singular they is better.

Anyway, I might need to remove the reference to this PR as this might be related to a bug instead of expected behavior, so it is not much important, at least for now.

Reply:
> Is the use of singular they really needed here? While I am not giving the name of the reporter, the reporter's identity is well known, I don't think this case needs a gender neutral form.

Fair question. I tend to err on the side of caution and use gender-neutral pronouns unless I know the person's preference, as the gender implied by the first name isn't 100% conclusive, non-binary people being an obvious case where it can mislead. That said, you appear to know the commenter better than I do (which is not at all), so I'll trust your judgement here.